Doc/library/sched.rst

```diff
@@ -20,8 +20,7 @@ scheduler:
    The :class:`scheduler` class defines a generic interface to scheduling events.
    It needs two functions to actually deal with the "outside world" --- *timefunc*
    should be callable without arguments, and return  a number (the "time", in any
```

Comment:
I dislike "should" in technical documentations. I suggest to replace "should be" with "is". Like "timefunc is a callback without parameter which returns a number (...)".

Reply:
I didn't write that snippet in the first place, but am glad to change it now 😉